src/client/java/distribution/pom.xml

```diff
@@ -30,6 +30,7 @@
               <descriptors>
                 <descriptor>src/assembly/make-assembly.xml</descriptor>
               </descriptors>
+	      <tarLongFileMode>posix</tarLongFileMode>
```

Comment:
Building the jar files is still failing. There are 3 different sections in this pom.xml
1. <descriptor>src/assembly/make-assembly.xml</descriptor>
2. <descriptor>src/assembly/make-assembly-with-dependencies.xml</descriptor>
3. <descriptor>src/assembly/make-assembly-with-proto3-netty4-deps.xml</descriptor>

The above change is for section 1. But `<tarLongFileMode>posix</tarLongFileMode>` should be added to those other 2 sections of pom.xml as well.
Specially for section 3 as we recommend building with `with-proto3-netty4-deps` in documentation i.e
`mvn -Pdistribute,with-proto3-netty4-deps clean package -DskipTests -Dgpg.skip -Ddaos.install.path=<daos_install>`

Reply:
Add `<tarLongFileMode>posix</tarLongFileMode>` after following lines as well:

https://github.com/daos-stack/daos/blob/c3bee59024a53a97a7943bf09a70c0237fbcaa7c/src/client/java/distribution/pom.xml#L58

https://github.com/daos-stack/daos/blob/c3bee59024a53a97a7943bf09a70c0237fbcaa7c/src/client/java/distribution/pom.xml#L83

Reply:
done